tools/bpf2c/bpf2c.cpp

```diff
@@ -129,15 +129,33 @@ main(int argc, char** argv)
         auto iter_end = parameters.end();
         std::map<std::string, std::tuple<std::string, std::function<bool()>>> options = {
             {"--sys",
```

Comment:
We might want to extend this string to read `--sys [out_file_name]` for a more helpful visual aid. (same for the other output types).

Reply:
<sigh> can't do it here.  maybe somehow incorporate this in the help text on the next line?